src/modules/Layout/Container/Container.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
```

Comment:
Just for a layout stuff. 
E.g. it easier to wrap some block with `<Container>` which will set alignment _css_ rules and if something will change we will just remove/replace/change-inside Container component and will not be forced to go to styles and rewrite some rules for some `.align-me-class` which could potentially affect some other component(in case if we were using `.align-me-class`.